runtime/cere/src/lib.rs

```diff
@@ -1317,6 +1318,71 @@ impl pallet_ddc_metrics_offchain_worker::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 }
 
+parameter_types! {
+	pub const DdcCustomersPalletId: PalletId = PalletId(*b"accounts"); // DDC maintainer's stake
+	pub const UnlockingDelay: BlockNumber = 5256000u32; // 1 hour * 24 * 365 = 1 day; (1 hour is 600 blocks)
```

Comment:
@aie0 @rakanalh 
`UnlockingDelay` - we need to set an unlocking delay that is good enough to prevent possible manipulate with debts and DAC cycle. This delay sets how long a customer should wait to unlock and transfer deposit he previously did

Reply:
we'll set to 7 days similar to optimistic rollup